WebContent/js/actions/treeUSS.js

```diff
@@ -172,6 +197,46 @@ export function createUSSResource(path, type) {
     };
 }
 
+export function createAndDownloadElement(blob, fileName) {
+    const elem = window.document.createElement('a');
+    elem.href = window.URL.createObjectURL(blob);
+    elem.download = fileName;
+    document.body.appendChild(elem);
+    elem.click();
+    document.body.removeChild(elem);
+}
+
+export function downloadUSSResource(path) {
+    return dispatch => {
+        dispatch(requestDownload(path));
+        const endpoint = `restfiles/fs/${path && path.indexOf('/') === 0 ? path.substring(1) : path}`;
+        const fileName = endpoint.substring(endpoint.lastIndexOf('/') + 1);
+        return atlasGet(endpoint, { credentials: 'include' })
+            .then(response => {
+                return dispatch(checkForValidationFailure(response));
+            })
+            .then(response => {
+                if (response.ok) {
+                    return response.text();
+                }
+                return response.text().then(e => { throw Error(JSON.parse(e).message); });
+            })
+            .then(text => {
+                const blob = new Blob([text], { type: 'text/plain' });
+                if (window.navigator.msSaveOrOpenBlob) {
```

Comment:
which browsers support this? I had trouble finding doc on it other than mozilla saying its deprecated and non-standard. is this a function actually supported by the browsers Zowe supports, or is this like, internet explorer?

Reply:
This was for IE . I just found that IE is not in the supported browsers for ZOWE, I removed it now. 
Thanks for review